react/features/filmstrip/components/native/ModeratorIndicator.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { IconModerator } from '../../../base/icons';
+// eslint-disable-next-line lines-around-comment
+// @ts-ignore
+import { BaseIndicator } from '../../../base/react';
+
+/**
+ * Thumbnail badge showing that the participant is a conference moderator.
+ *
+ * @returns {JSX.Element}
+ */
+export default function MoeratorIndicator(): JSX.Element {
```

Comment:
typo in `MoeratorIndicator`

Reply:
also do we need named function here?

Reply:
Not necessarily, it just serves as "documentation".